src/setupTest.js

```diff
@@ -9,13 +9,16 @@ import { configure as configureI18n } from '@edx/frontend-platform/i18n';
 import { configure as configureLogging, MockLoggingService } from '@edx/frontend-platform/logging';
 import { getConfig, mergeConfig } from '@edx/frontend-platform';
 import { configure as configureAuth, MockAuthService } from '@edx/frontend-platform/auth';
+import envConfig from '../env.config';
 import messages from './i18n';
 
 jest.mock('@edx/frontend-platform/react/hooks', () => ({
   ...jest.requireActual('@edx/frontend-platform/react/hooks'),
   useTrackColorSchemeChoice: jest.fn(),
```

Comment:
To answer this question and why it was duplicated in the first place (a question I had):
1. I _don't_ know what it's used for, but it was originally mocked in [May 2023 during the Atlas translations work](https://github.com/openedx/frontend-app-learner-record/pull/152/files) that AXIM was implementing. My understanding is that the upgrade to Node 18 was force-pushed in this PR as well, but that upgrade didn't need the mock. That said, without the mock, all tests will fail 🤷🏽 @OmarIthawi can you recall how you knew to mock this specific hook? 
2. The second time it was mocked was during a [PR in September '23 while updating React Router to v6](https://github.com/openedx/frontend-app-learner-record/pull/150)

From what I understand, `useTrackColorSchemeChoice` will be called for every MFE that uses `AppProvider` [out of the box](https://github.com/openedx/frontend-platform/blob/1d3e6bbc3b8c718c3ef08bb91528bb39d10d1aaa/src/react/AppProvider.jsx#L52) from `frontend-platform`, and [that method's](https://github.com/openedx/frontend-platform/blob/1d3e6bbc3b8c718c3ef08bb91528bb39d10d1aaa/src/react/hooks.js#L32) `useTrackEvent` has been the sole culprit for why the tests fail. I'm not sure if the clearer solution would be to mock `useTrackEvent` in this case, since it is still not directly called in this set up.

Reply:
Ah, OK... I see, this MFE (and others) actually render `AppProvider` in its tests, which results in `useTrackColorSchemeChoice` being called; other MFEs are following a similar convention. Whereas typically in the Enterprise MFEs, we don't render `AppProvider` itself in any tests, so we haven't seen this issue arise before.

Also just tried to play around with your branch, and came to the same conclusion that if rendering the actual `AppProvider` component as part of the test, then this would indeed need to be mocked in some way.

Thanks for clarifying!

Reply:
Please don't trust the mock I made here. I mostly copied it from other MFEs to get through the `frontend-platform` upgrade. If the mock isn't needed, it has absolutely no use to Atlas or translations as long as `frontend-platform` newer than `v4.2.0` i18n workflows are okay.

Anyway, it seems that you've already resolved the issue, which is good to hear!